lib/recipe.rb

```diff
@@ -143,20 +179,22 @@
 esac
 INIT_SCRIPT
 
-file init_script_path do
-  content(init_script)
-  mode 0755
-  owner 'root'
-end
+  file init_script_path do
```

Comment:
The whole block is on the else side if an if platform?('windows'); do you think we need to have both?